src/applications/timeouts.ts

```diff
@@ -2,10 +2,25 @@ import { assign } from "../utils/assign";
 import { getProps } from "../lifecycles/prop.helpers";
 import { objectType, toName } from "./app.helpers";
 import { formatErrorMessage } from "./app-errors";
+import { AppOrParcel } from "../lifecycles/lifecycle.helpers";
+
+export interface AppOrParcelTimeouts {
```

Comment:
The AppOrParcelTimeouts and Timeout types were moved from lifecycle.helpers.ts, since it seemed more relevant to timeouts.ts